### ISSUE TITLE
is.numeric(width) error

### ISSUE THREAD

**Issue by rajesh06** [d-qn/2016_08_02_rioOlympicsAthletes#2]
I get a 
`Error: is.numeric(width) is not TRUE` 
error with the original code which is corrected by changing width = "100%" to width = 1 in Line 175

**Comment by d-qn** [comment]
Yes, thanks for that. I ran an older version of ggiraph where the width had to be provided in CSS unit. With the latest ggiraph, width has indeed to be a numeric between 0 and 1
